README.md

```diff
@@ -129,23 +129,16 @@ bot's token. You must also depend on `futures`, `tokio`,
 use std::{env, error::Error, sync::Arc};
 use futures::stream::StreamExt;
 use twilight_cache_inmemory::{InMemoryCache, ResourceType};
-use twilight_gateway::{cluster::{Cluster, ShardScheme}, Event};
+use twilight_gateway::{Cluster, Event};
 use twilight_http::Client as HttpClient;
 use twilight_model::gateway::Intents;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     let token = env::var("DISCORD_TOKEN")?;
 
-    // This is the default scheme. It will automatically create as many
-    // shards as is suggested by Discord.
-    let scheme = ShardScheme::Auto;
```

Comment:
Considering this is in our most visible README I believe we should continue to document what exactly a Cluster is. Even just one sentence would do

Reply:
This makes sense